config/locales/frequencies.yml

```
@@ -1,5 +1,5 @@
 en:
   frequencies:
-    immediately: As soon as they happen
-    daily: Once a day
-    weekly: Once a week
+    immediately: Every time something changes on the GOV.UK website
```

The [content doc](https://docs.google.com/document/d/1eo_BTlA0EBVDabz1tK3W1wIIEqB4SlsjM0fGQnKCZYE/edit) suggests this should be:

```suggestion
    immediately: Every time something changes on GOV.UK
```